components/accept-financial-contributions/ApplyToHost.js

```diff
@@ -125,8 +125,13 @@ class ApplyToHost extends React.Component {
             px={[3, 5]}
             my={3}
           >
+<<<<<<< HEAD
```

Comment:
@ericyoungson : Same as above the merge from main branch probably didn't go well. You'll need to resolve these conflicts. 🤔